### ISSUE TITLE
[general] locale capture is a bit broken

### ISSUE THREAD

**Issue by bmr-cymru** [sosreport/sos#409]
We now consistently use the C locale when running commands which is a `good thing` but causes:

```
$ cat sosreport-atomicdockercontainer-20140925070217/sos_commands/i18n/locale 
LANG=
LC_CTYPE="C"
LC_NUMERIC="C"
LC_TIME="C"
LC_COLLATE="C"
LC_MONETARY="C"
LC_MESSAGES="C"
LC_PAPER="C"
LC_NAME="C"
LC_ADDRESS="C"
LC_TELEPHONE="C"
LC_MEASUREMENT="C"
LC_IDENTIFICATION="C"
LC_ALL=C
```

**Comment by TurboTurtle** [comment]
@bmr-cymru we could either solve this by simply setting `env = {'LC_ALL': ''}` in the `i18n` plugin, or we could add a mechanism to `sos_get_command_output()` that would allow any call to `add_cmd_output()` to _remove_ a particular env var that gets handed to `Popen()`.

Aside from this exact issue, I'm not coming up with anything as a use-case for the latter - just a thought.

**Comment by bmr-cymru** [comment]
It's a corner case, for sure - I'm hovering between either allowing plugins to specify the _whole_ environment, or as you suggest, filtering out certain bits. I think I need to think about it a bit more, but in the meantime happy to review any suggestions or ideas.

**Comment by TurboTurtle** [comment]
Resolved with #1362
